app/scripts/lib/experiments/choices/communication-prefs.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const BaseExperiment = require('./base');
```

Comment:
nit: `BaseExperiment` should probably be called `BaseChoice` or `BaseChoiceRule`